src/Test/Perf/util/ITraceManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+namespace Roslyn.Test.Performance.Utilities
+{
+    public interface ITraceManager
```

Comment:
`ITraceManager` no longer has this interface. There is no `Iterations`. There are `HasWarmupIterations` property and `Initialize` method